tests/unit/Alteration.spec.ts

```diff
@@ -284,6 +325,52 @@ describe('Alteration component', () => {
     // Validate Contact Info
     expect(store.state.stateModel.defineCompanyStep.businessContact.email).toBe('mock@email.com')
     expect(store.state.stateModel.defineCompanyStep.businessContact.phone).toBe('123-456-7890')
+
+    expect(store.state.stateModel.currentFees.filingFees).toBe(100)
+    expect(store.state.stateModel.currentFees.futureEffectiveFees).toBe(0)
+  })
+
+  it('fetches the fee prices after loading', async () => {
+    await wrapper.setProps({ appReady: true })
+    await flushPromises()
+    expect(store.state.stateModel.feePrices.filingFees).toBe(100)
+    expect(store.state.stateModel.feePrices.futureEffectiveFees).toBe(100)
+  })
+
+  it('display the fee prices properly', async () => {
+    await wrapper.setProps({ appReady: true })
+    await flushPromises()
+    store.state.stateModel.summaryMode = true
+    store.state.stateModel.tombstone.filingType = FilingTypes.ALTERATION
```

Comment:
Don't use enums in the tests. Use the actual value. This makes the tests more fragile, so they will fail if a developer updates the enum without updating the tests.

Reply:
ok.